pkg/cmd/server/apis/config/v1/types.go

```diff
@@ -1420,6 +1420,21 @@ type ServiceServingCert struct {
 	Signer *CertInfo `json:"signer"`
 }
 
+// UnidlingProxyConfig holds configuration for the unidling network proxy, which triggers unidling
+// on the presence of network traffic.
+type UnidlingProxyConfig struct {
+	// Enabled controls whether or not the unidling proxy is enabled.
+	Enabled bool `json:"enabled"`
+
+	// NFQueueNumber is the number of the NFQueue used to intercept unidling traffic.
+	// It must be unique.
+	NFQueueNumber uint16 `json:"nfQueueNumber"`
```

Comment:
These are incredibly specific config values - I'm not sure a human should ever set them.

Reply:
Well, specifically, it needs to never conflict with the two Kubernetes mark bits (one is the masquerade bit, and the other is similiar), and it needs to not conflict with any other rules people set on their systems -- that's why it's configurable.

Reply:
Again, this is not something a human should have to set ever.  Can you determine values for this  for 99% of cases that won't conflict?